content/projects/kauri/index.md

```diff
@@ -0,0 +1,82 @@
+author: "Paul Sanderson"
+description: "Kauri is an open-source, multi-currency wallet that will be the foundation for advanced features like mobile staking, merchant tools, atomic exchanges and many more great NavCoin features."
+resources: []
+categories: []
+filter: ["Wallets"]
+bg_image: "/images/icons/icon-wallet-purple-large.png"
+slug: ""
+type: "project"
+feature_image: ""
+show_title_text: false
+in_progress: true
+completed: false
+show_on_roadmap: true
+show_on_projects: true
+allow_click_through: true
+percent_complete: 10
+github_url: "https://github.com/Encrypt-S/kauri-wallet"
+project_url: ""
+reddit_url: ""
+twitter_url: ""
+card_color: "#6984D8"
+font_color: "#fff"
+weight: 0
+---
+{{< project_header
+    title="Kauri Wallet"
+    subhead=""
+    gradient=""
+    text="Kauri gives your crypto wallet the features you expect from your banking app like accounts and recurring payments but then so much more."
+    captionPadding="100px"
+    imgSrc="/images/icons/nav_purple-ecofriendly-large.png"
+    imgHeight="300px"
+    imgPadding="10px"
+    bgColor="#FFF"
+    fontColor="#333">}}
+{{< center_text_section
+    title="It’s open source"
+    bgColor="#42b6eb"
+    gradient=""
+    fontColor="#FFF">}}
+    <p>Transparency is an important pillar of decentralised software, especially when trusting an application to store your cryptocurrency. The Kauri Wallet will always be free and open source, so you can rest easy knowing the software used to secure your funds can be validated by anyone at anytime.</p>
+{{< /center_text_section >}}
+{{< project_features
+    fontColor="#333"
+    gradient="false"
+    feature1Img="/images/icons/nav_blue-wallet-large.png"
+    feature1Title="Multi-Currency"
+    feature1Txt="Kauri is designed to work with multiple blockchains to enable all the advanced NavCoin features Encrypt S is building for your entire portfolio."
+    feature2Img="/images/icons/nav_blue-greentick-large.png"
+    feature2Title="Multi-Platform"
+    feature2Txt="Kauri can be used on desktop, mobile and web browser with the same advanced features and seamless user experience."
+    feature3Img="/images/icons/nav_blue-weeklyupdate-large.png"
+    feature3Title="Multi-Purpose"
+    feature3Txt="Kauri will be able to operate as a thin client, full node or a staking node either on your desktop, mobile or remotely from a Raspberry Pi."
+    feature4Img="/images/icons/nav_blue-peertopeer-large.png"
+    feature4Title="Kauri DAEx"
+    feature4Txt="Kauri will be like having a decentralised atomic exchange in your pocket. It enables safe peer 2 peer exchanges for any supported currency."
+    feature5Img="/images/icons/nav_blue-merchants-large.png"
+    feature5Title="NavDelta"
+    feature5Txt="Kauri will have a suite of merchant tools including a payment gateway that enables users to spend NAV at any business which accepts a Kauri supported currency."
+    feature6Img="/images/icons/icon-network.png"
+    feature6Title="Nav Morph"
```

Comment:
This shouldn't have a space in it right?
Also the icon being used here is low resolution compared to the others, really stands out to MacBook users